src/lib/data/aligned-text.js

```diff
@@ -0,0 +1,49 @@
+import Token from '@/lib/data/token'
+import TokenizeController from '@/lib/controllers/tokenize-controller.js'
+
+export default class AlignedText {

```

Comment:
I still think ow to realize segments, because from requirements:
we should have dependency between parts of origin and translation texts - I understand it this way, that we should be sure that a segment from originText should be shown and scrolled with the same part of translation text.
Then both should have depended segments,
@balmas , am I right?
I suggest , by the way to move this discussin to the issue , I created

Reply:
It's for more than scrolling, the segments also have a semantic meaning that needs to be preserved per this requirement: 

_Must support creation of a new alignment via upload of a set of plain text matched segments_

But agree we can discuss in #27.